uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/leave-request-instance_test.js

```diff
@@ -224,7 +233,17 @@ define([
         promise.then(function () {
           expect(LeaveRequestInstance.toAPI).toHaveBeenCalled();
         });
-      })
+      });
+
+      it('calls API to save the newly created comments only(without having ID)', function () {
```

Comment:
`calls API to save the newly created comments only which don't have ID`
or something similar